Marlin/src/lcd/extui/lib/anycubic_chiron/chiron_tft.cpp

```diff
@@ -305,7 +305,8 @@ namespace Anycubic {
         SERIAL_EOL();
       #endif
     }
-    TFTSer.println("");
+    TFTSer.write('\r');
+    TFTSer.write('\n');
```

Comment:
Wouldn't `TFTSer.println()` be enough here?

Reply:
It used to work ok like that but the recent changes to the core serial comms broke the behaviour and it stopped working.

Reply:
Got it. My question is why you don't use `TFTSer.println();` here as it's doing exactly the same thing as `TFTSer.write(\r); TFTSer.write(\n);` ?

Reply:
So the empty quotes messed it up?
I'll try it now

Reply:
Yes 
`TFTSer.println();`works ok

The original `TFTSer.println("");` doesn't.

Reply:
Strange. I'll double check this.

Reply:
Can you send me your firmware.elf with the `println("")` that does not work ? I'd like to check why it did not build as expected.

Reply:
Sorry my bad,
I reverted too many changes it was the `write(c)` change in the lines above that caused the issue.

Reply:
Still it's better to have the version without the string (println()). Thanks for pointing the error.